api/proto/disperser/disperser.proto

```diff
@@ -97,18 +97,25 @@ message DisperseBlobReply {
 	// The status of the blob associated with the request_id. Will always be PROCESSING.
 	BlobStatus result = 1;
 	// The request ID generated by the disperser.
-	// Once a request is accepted (although not processed), a unique request ID will be
-	// generated.
-	// Two different DisperseBlobRequests (determined by the hash of the DisperseBlobRequest)
-	// will have different IDs, and the same DisperseBlobRequest sent repeatedly at different
-	// times will also have different IDs.
-	// The client should use this ID to query the processing status of the request (via
-	// the GetBlobStatus API).
+	//
+	// Once a request is accepted, and before it is processed, a unique request ID is generated.
```

Comment:
`and before it is processed` can be dropped I think

Reply:
[79a23b7](https://github.com/Layr-Labs/eigenda/pull/1360/commits/79a23b76429cc52aeb49d814b00759abe8c0010a)